astronomer/providers/google/cloud/operators/dataproc.py

```diff
@@ -172,61 +51,20 @@ def __init__(
         polling_interval: float = 5.0,
         **kwargs: Any,
     ):
-        super().__init__(**kwargs)
-        self.polling_interval = polling_interval
-        if self.timeout is None:
-            self.timeout: float = 24 * 60 * 60
-
-    def execute(self, context: Context) -> None:
-        """Call delete cluster API and defer to wait for cluster to completely deleted"""
-        hook = DataprocHook(gcp_conn_id=self.gcp_conn_id, impersonation_chain=self.impersonation_chain)
-        self.log.info("Deleting cluster: %s", self.cluster_name)
-        hook.delete_cluster(
-            project_id=self.project_id,
-            region=self.region,
-            cluster_name=self.cluster_name,
-            cluster_uuid=self.cluster_uuid,
-            request_id=self.request_id,
-            retry=self.retry,
-            metadata=self.metadata,
-        )
-
-        try:
-            hook.get_cluster(project_id=self.project_id, region=self.region, cluster_name=self.cluster_name)
-        except NotFound:
-            self.log.info("Cluster deleted.")
-            return
-        except Exception as e:
-            raise AirflowException(str(e))
-
-        end_time: float = time.time() + self.timeout
-
-        self.defer(
-            trigger=DataprocDeleteClusterTrigger(
-                gcp_conn_id=self.gcp_conn_id,
-                project_id=self.project_id,
-                region=self.region,
-                cluster_name=self.cluster_name,
-                request_id=self.request_id,
-                retry=self.retry,
-                end_time=end_time,
-                metadata=self.metadata,
-                impersonation_chain=self.impersonation_chain,
+        warnings.warn(
+            (
+                "This module is deprecated and will be removed in 2.0.0."
+                "Please use `airflow.providers.google.cloud.operators.dataproc.DataprocDeleteClusterOperator`"
+                "and set `deferrable` param to `True` instead."
             ),
-            method_name="execute_complete",
+            DeprecationWarning,
+            stacklevel=2,
         )
+        super().__init__(deferrable=True, **kwargs)
```

Comment:
Look like we need to add "polling_interval_seconds" in kwargs here too, no?
https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/operators/dataproc.py#L1002

Reply:
yep, miss this one. just updated it. thanks!